packages/flutter_webrtc/README.md

```diff
@@ -0,0 +1,47 @@
+# flutter_webrtc_tizen
+
+[![pub package](https://img.shields.io/pub/v/flutter_webrtc_tizen.svg)](https://pub.dev/packages/flutter_webrtc_tizen)
+
+The Tizen implementation of [`flutter_webrtc`](https://github.com/flutter-webrtc/flutter-webrtc).
+
+## Required privileges
+
+To use this plugin in a Tizen application, you may need to declare the following privileges in your `tizen-manifest.xml` file.
+
+```xml
+<privileges>
+    <privilege>http://developer.samsung.com/privilege/camera</privilege>
+    <privilege>http://tizen.org/privilege/internet</privilege>
+    <privilege>http://tizen.org/privilege/camera</privilege>
+    <privilege>http://tizen.org/privilege/recorder</privilege>
+</privileges>
+```
+
+- The internet privilege (`http://tizen.org/privilege/internet`) is required to access the internet.
+- The camera priviege (`http://tizen.org/privilege/internet`) is required to use camera.
+- The recorder priviege (`http://tizen.org/privilege/recorder`) is required to record video and audio.
+
+ To use camera on Tizen TV devices, you need to add developer camera privilege (`http://developer.samsung.com/privilege/camera`) and have a [partner level certificate](https://developer.samsung.com/tv-seller-office/guides/membership/becoming-partner.html).
```

Comment:
The link is a bit misleading. As far as I know "becoming a partner" and "acquiring a partner-level certificate" do not have the same meaning. We may need a different URL or a better description.

Reply:
As far as I know, we need to because a partner seller if we want to use 'http://developer.samsung.com/privilege/camera'
I have modified to 'become a [partner seller]'

Reply:
Isn't `http://developer.samsung.com/privilege/camera` a partner-level privilege? Then a user has to sign their TPK with a partner-level certificate and the original description was valid. What I'm talking about is the link, which only explains how to "become a partner" but not "how to acquire a partner-level certificate".

Reply:
I update Uri with [partner level certificate](https://docs.tizen.org/application/dotnet/get-started/certificates/creating-certificates)

Reply:
The change looks good, but when I issue a partner-level certificate by following the instructions and try to install a TPK signed with the certificate on a TV device, I get the following error. Can you reproduce the error? How did you create your certificate?

```sh
path is /tmp/com.example.example-1.0.0.tpk
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[start] val[install]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[10]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[13]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[16]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[18]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[21]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[24]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[install_percent] val[27]
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[error] val[-12] error message: :Invalid format of certificate in signature.:<-2>
__return_cb req_id[1] pkg_type[tpk] pkgid[com.example.example] key[end] val[fail]
processing result : Check certificate error [-12] failed
```

Reply:
I follow this guide to create [partner certificate](http://wiki.vd.sec.samsung.net/pages/viewpage.action?spaceKey=VDSecurity&title=0.+Certificate+for+internal+development+testing)

Reply:
I have your issue, too. build tpk after starting the computer.
In this case, you need to open certificate manager and check certificate profile.

Reply:
That page is only for internal developers. 3rd-party developers should be able to issue their own certificates and use them for app development by only following this README.

Reply:
Yes, this only for internal developer. 
3rd-party developers can't use plugin on TV devices，only partner can access camera on TV device.

Reply:
I mean, partners are also third-party developers. We need to ensure that partner-level certificates issued by Tizen Studio can be used for TV app development outside the company.

Reply:
Hmm. This needs to be checked. It is also possible that an invalid certificate is being created by Tizen Studio or the partner-level certificate cannot be used with debug images.

@xuelian-bai Do you know any details? Should a developer always request a partner-level certificate (e.g. via email) to test their app during development?

Reply:
Okay. Let me check this later once I get a new TV and then I will update the document.

Reply:
I managed to install the example app on my brand new TV (S27BM502) by signing the package with a partner-level certificate that I issued just a minute ago using Certificate Manager.

I couldn't test the app because the TV doesn't have a camera installed but let's merge the PR anyway.